sinch/domains/voice/__init__.py

```diff
@@ -0,0 +1,180 @@
+from sinch.domains.voice.endpoints.callouts.callout import CalloutEndpoint
+from sinch.domains.voice.endpoints.calls.get_call import GetCallEndpoint
+from sinch.domains.voice.endpoints.calls.update_call import UpdateCallEndpoint
+from sinch.domains.voice.enums import CalloutMethod
+from sinch.domains.voice.models.callouts.responses import VoiceCalloutResponse
+from sinch.domains.voice.models.callouts.requests import (
+    ConferenceVoiceCalloutRequest,
+    TextToSpeechVoiceCalloutRequest,
+    CustomVoiceCalloutRequest
+)
+from sinch.domains.voice.models.calls.requests import GetVoiceCallRequest, UpdateVoiceCallRequest
+from sinch.domains.voice.models.calls.responses import GetVoiceCallResponse
+
+
+class Callouts:
+    def __init__(self, sinch):
+        self._sinch = sinch
+
+    def text_to_speech(
```

Comment:
Name to be validated: `textToSpeech` in Java SDK, `Tts` in .NET and `tts` in Node.js

Reply:
Personally I prefer `text_to_speech`